packages/ui/src/components/Select/SSelectBase.vue

```diff
@@ -52,10 +52,20 @@ const props = withDefaults(
      */
     sameWidthPopper?: boolean
 
+    /**
+     * Adds search field to trigger
+     */
+    triggerSearch?: boolean
+
     /**
      * Adds search field to dropdown
      */
-    dropdownSearch: boolean
+    dropdownSearch?: boolean
+
+    /**
+     * Turns off options filtering by select itself. When false select filters by option's label. Adds chips
```

Comment:
```suggestion
     * By default, the component filters options by their labels. Set `true` to disable automatic filtering.
```

What means "Add chips"? There are no any chips functionality in this component, so I am confused.

Reply:
I'll remove mention of the chips because it SSelect design details

Reply:
changed